Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceConfigurationPolicyAndroidDeviceAdministrator/MSFT_IntuneDeviceConfigurationPolicyAndroidDeviceAdministrator.schema.mof

```diff
@@ -0,0 +1,90 @@
+    [Write, Description("Kiosk mode apps"), EmbeddedInstance("MSFT_MicrosoftGraphapplistitem")] String KioskModeApps[];
+    [Write, Description("Kiosk mode block sleep button")] Boolean KioskModeBlockSleepButton;
+    [Write, Description("Kiosk mode block volume buttons")] Boolean KioskModeBlockVolumeButtons;
+    [Write, Description("Location services blocked")] Boolean LocationServicesBlocked;
+    [Write, Description("Block Near Field Communication (NFC) technology (Samsung KNOX Standard 4.0+).")] Boolean NfcBlocked;
+    [Write, Description("Block using fingerprint to unlock device.")] Boolean PasswordBlockFingerprintUnlock;
+    [Write, Description("Block Smart Lock or other trust agents from adjusting lock screen settings (Samsung KNOX Standard 5.0+).")] Boolean PasswordBlockTrustAgents;
+    [Write, Description("Number of days until device password must be changed. (1-365)")] UInt32 PasswordExpirationDays;
+    [Write, Description("Minimum number of digits or characters in password. (4-16)")] UInt32 PasswordMinimumLength;
+    [Write, Description("Maximum minutes of inactivity until screen locks. Ignored by device if new time is longer than what's currently set on device. If set to Immediately, devices will use the minimum possible value per device.")] UInt32 PasswordMinutesOfInactivityBeforeScreenTimeout;
+    [Write, Description("Number of new passwords that must be used until an old one can be reused.")] UInt32 PasswordPreviousPasswordBlockCount;
+    [Write, Description("Require password to access device.")] Boolean PasswordRequired;
+    [Write, Description("Specify the type of password required."), ValueMap{"deviceDefault","alphabetic","alphanumeric","alphanumericWithSymbols","lowSecurityBiometric","numeric","numericComplex","any"}, Values{"deviceDefault","alphabetic","alphanumeric","alphanumericWithSymbols","lowSecurityBiometric","numeric","numericComplex","any"}] String PasswordRequiredType;
+    [Write, Description("Number of consecutive times an incorrect password can be entered before device is wiped of all data.")] UInt32 PasswordSignInFailureCountBeforeFactoryReset;
+    [Write, Description("Block user from powering off device. If this setting is disabled the setting 'Number of sign-in failures before wiping device' does not function.")] Boolean PowerOffBlocked;
+    [Write, Description(""), ValueMap{"none","low","medium","high"}, Values{"none","low","medium","high"}] String RequiredPasswordComplexity;
+    [Write, Description("Block capturing contents of screen as an image.")] Boolean ScreenCaptureBlocked;
+    [Write, Description("Security require verify apps")] Boolean SecurityRequireVerifyApps;
+    [Write, Description("Block sync with Google backup.")] Boolean StorageBlockGoogleBackup;
+    [Write, Description("Block removable storage usage (Samsung KNOX Standard 4.0+).")] Boolean StorageBlockRemovableStorage;
+    [Write, Description("Require encryption on device. Not all devices support encryption.<br>
```

Comment:
Can we please make the description on one line and remove the HTML tags even if that may make it harder to read? Thanks